docker/nginx/conf.d/client.conf

```diff
@@ -480,19 +484,21 @@ server {
 		}
 
 		proxy_set_header Dnslink-Lookup "";
-		proxy_pass http://127.0.0.1/$dnslink/$request_uri;
+		proxy_pass http://127.0.0.1/$dnslink$request_uri;
```

Comment:
Do we ever verify that the `dnslink` ends with a slash? I see we're grabbing it from the headers in 
```
ngx.var.dnslink = headers["skynet-skylink"]
```
but I'm not sure we can rely on that being always valid.

Can we verify that it's either empty or ends with a slash?

Reply:
it's not that $dnslink ends with slash, it's $request_uri that always starts with one :)